testing/live/DeeperDeepDungeonDex/manifest.toml

```diff
@@ -0,0 +1,5 @@
+[plugin]
+repository = "https://github.com/NotNite/DeeperDeepDungeonDex.git"
+commit = "6e96fbb599fbb4153222f6a7642738c4cbfaf61f"
+owners = ["NotNite", "MidoriKami"]
+project_path = "DeeperDeepDungeonDex"
```

Comment:
**blocking;**
while i acknowledge the entertainment value of the name, pac is on the whole not comfortable with merging this as-stands, using a name that is clearly riffing on another plugin - regardless of circumstances.
i'm not going to ask yall to go through refactoring the entire plugin over this, but i _will_ ask you update user-facing strings, please.

Reply:
I'll state upfront, I am willing to change the name to `[anything]DeepDungeonDex`.

WildWolf stated quite publicly that he intended to change the name of `DeepDungeonDex` (This can be seen in a public channel here: [XIVLauncherDiscord#DeepDungeonDex](https://discord.com/channels/581875019861328007/1019651246619099136/1195346556572930108)

By having some form of `DeepDungeonDex` in the name of this plugin then users that are interested only in Deep Dungeon content could easily find our plugin.

If this is insufficient then I would like to ask for a more real-time dialogue with PAC to discuss options.

Reply:
heyo, i think the main issue we want to address is the direct inclusion of the existing plugin's name. we're by no means fortune tellers, but the potential for confusion or conflict over this is deemed undesireable.

there's no concern held with inclusion of "deep dungeon" in the title - as you say, users may search by content name, and including it in the title is valuable. the ask is _only_ to avoid the direct inclusion of "DeepDungeonDex" as-stands. some off-the-cuff names pac threw around at the time include "Deep Dungeon Helper" and "Of Dungeons Deep" (lmao i love it), but these are just some ideas.

while i doubt there's much to be gained from real time conversations bar reiteration of the above, by all means do reach out if you'd like to run ideas past - it's unlikely i'll specifically be around directly due to availability/timezones, but i'm fairly sure there'll be someone at most times.

it looks like someone's come through with a testing pass, so this looks good to go once naming is addressed

Reply:
This is acceptable. ODD will work.